mne/viz/ica.py

```diff
@@ -63,6 +64,8 @@ def plot_ica_sources(ica, inst, picks=None, exclude=None, start=None,
         Whether to halt program execution until the figure is closed.
         Useful for interactive selection of components in raw and epoch
         plotter. For evoked, this parameter has no effect. Defaults to False.
+    ignore_first_samp : bool
+        If False, show time axis relative to the ``raw.first_samp``.
```

Comment:
Double negatives are tough, how about `show_first_samp=False` by default, and `show_first_samp=True` means show times relative to `first_samp`?